examples/bpk-component-price/stories.js

```diff
@@ -17,44 +17,41 @@
  */
 
 import {
-  SmallLeftExample,
```

Comment:
why were all these renamed? If I remember correctly, we use `Example` at the end of the name for all example components

Reply:
This was due to some of them getting really long and I noticed we don't fully follow this pattern all the way through with all our examples so I thought this was ok, especially with an example name of

`SmallLeftWithTrailingAndLeadingText`

Happy to change it though if you would like and we want to make it official all stories should change to this naming

Reply:
Although now I do notice I switched it around the naming of the examples and the stories, so let me switch that back to add them at the end.

They will still be renamed but not remove `Example`